src/pages/SettingsPage/UsersSettings/UserList.jsx

```diff
@@ -41,8 +52,8 @@ const UserList = ({
   }
 
   // IDEA: Can these go into the details panel as well?
-  const ctxMenuTableItems = useMemo(
-    () => [
+  const ctxMenuItems = () => {
```

Comment:
The selection is passed to ctxMenuItems on like 44 but it's never used here.

`onDelete` has a bug where selecting one user and then right clicking on another and then deleting will delete the old selection and not the new one.

See how user35 is the one being deleted even though we right clicked on user33.

To fix this pass the newSelection to `onDelete` and use that selection argument on UserSettings.jsx onDelete function.


https://github.com/user-attachments/assets/a72140fa-dabb-426e-9d30-d8baea78545c